.env-cmdrc

```diff
@@ -1,9 +1,9 @@
 {
   "dev": {
-    "CX_BASE_URL": "https://40.76.109.9:9002"
+    "CX_BASE_URL": "https://spartacus-devci7672.eastus.cloudapp.azure.com:9002"
   },
   "ci": {
-    "CX_BASE_URL": "https://20.83.184.244:9002"
+    "CX_BASE_URL": "https://spartacus-devci7672.eastus.cloudapp.azure.com:9002"
```

Comment:
We should change this back to the LB address before merging, right?

Reply:
yes, correct.

Reply:
done